src/components/BannerAlert/WarningBanner/index.tsx

```diff
@@ -29,7 +29,7 @@ class WarningBannerAlert extends React.PureComponent<Props, State> {
 
     return (
       <Banner className="pf-u-text-align-center" variant="warning">
-        {warningMessage}
+        <div dangerouslySetInnerHTML={{ __html: warningMessage }}></div>
```

Comment:
I'm OK with that as simplest option to go, assuming that the cluster admin is responsible for it's content. Then this is not more dangerous than have an ability to modify css, scripts on the packaging, or directly when you do kubectl exec )

Reply:
Well, later we will introduce an easy option to modify it for Che, CRW on CustomResource instance, but again, everybody who is able edit CheCluster CR - could do anything with custom dockerimage.

Reply:
We could sanitize the message from all tags except those we want to permit, using library like this one: https://www.npmjs.com/package/sanitize-html-react